Documentation/gettingstarted/k8s-install-azure.rst

```diff
@@ -102,3 +104,14 @@ Limitations
 
 * All VMs and VM scale sets used in a cluster must belong to the same resource
   group.
+
+.. _azure_troubleshooting:
+
+Troubleshooting
+===============
+* If ``kubectl exec`` to a pod fails to connect, restarting the ``tunnelfront`` pod may help.
+* Pods may fail to gain a ``.spec.hostNetwork`` status even if restarted and managed by Cilium.
+* If some connectivity tests fail to reach the ready state you may need to restart the unmanaged pods again.
+* Some connectivity tests may fail. This is being tracked in `Cilium GitHub issue #12113
+  <https://github.com/cilium/cilium/issues/12113>`_.
+* ``hubble observe`` may report one or more nodes being unavailable and ``hubble-ui`` may fail to connect to the backends.
```

Comment:
Do we need to file an issue for this one too?

Reply:
Not sure, most likely it is due to the more general connectivity issue rather than specific to hubble.

Reply:
I see. Is there something the user can do about this or we're just trying to let them know this is normal behaviour or something else?

Or is this more detail for the previous point? Because currently it's a standalone bullet point under "troubleshooting" that just says "hubble observe may report nodes being unavailable", but doesn't explain the context, whether that's normal, what might help to mitigate this issue, etc.

Reply:
using hubble is a separate step in the GSG, just wanted to note that as-of now it likely does not work. I'm not sure if this is due to the previous point or not, so can't tell if this is more detail on the previous point or not. It likely is, though, so maybe make this a sub-bullet?

Reply:
sounds good, let's make it a sub-bullet. We can re-evaluate when 12113 is addressed.

Reply:
Done.